src/wayland/dmabuf/mod.rs

```diff
@@ -269,8 +275,7 @@ impl<H: DmabufHandler> ParamsHandler<H> {
         plane_idx: u32,
         offset: u32,
         stride: u32,
-        modifier_hi: u32,
-        modifier_lo: u32,
+        modifier: (u32, u32),
```

Comment:
is it completely obvious, whats high and low, or should we add a little comment here?
Also below you are only using this as a u64. Is this two part really necessary?

Reply:
The two parts are how it is transmitted from the protocol. But maybe the conversion can be done before this function, indeed.